nginx-autoinstall.sh

```diff
@@ -189,20 +189,20 @@ case $OPTION in
 
 	# Dependencies
 	apt-get update
-	apt-get install -y build-essential ca-certificates wget curl libpcre3 libpcre3-dev autoconf unzip automake libtool tar git libssl-dev zlib1g-dev uuid-dev lsb-release libxml2-dev libxslt1-dev cmake
-
+	apt-get install -qy apt-utils m4 build-essential ca-certificates wget curl libpcre3 libpcre3-dev autoconf unzip automake libtool tar git libssl-dev zlib1g-dev uuid-dev lsb-release libxml2-dev libxslt1-dev cmake
+	apt-get update
+	
 	if [[ $MODSEC == 'y' ]]; then
 		apt-get install -y apt-utils libcurl4-openssl-dev libgeoip-dev liblmdb-dev libpcre++-dev libyajl-dev pkgconf
 	fi
 
 	# PageSpeed
 	if [[ $PAGESPEED == 'y' ]]; then
 		cd /usr/local/src/nginx/modules || exit 1
-		wget https://github.com/pagespeed/ngx_pagespeed/archive/v${NPS_VER}-stable.zip
-		unzip v${NPS_VER}-stable.zip
-		cd incubator-pagespeed-ngx-${NPS_VER}-stable || exit 1
-		psol_url=https://dl.google.com/dl/page-speed/psol/${NPS_VER}.tar.gz
-		[ -e scripts/format_binary_url.sh ] && psol_url=$(scripts/format_binary_url.sh PSOL_BINARY_URL)
+		wget https://github.com/apache/incubator-pagespeed-ngx/archive/v1.14.33.1-RC1.zip
+		unzip v1.14.33.1-RC1.zip
+		cd incubator-pagespeed-ngx-1.14.33.1-RC1 || exit 1
+		psol_url=https://dist.apache.org/repos/dist/release/incubator/pagespeed/1.14.36.1/x64/psol-1.14.36.1-apache-incubating-x64.tar.gz
```

Comment:
Please use `NPS_VER` instead of hardcoding the version and use the same version (`v1.14.33.1-RC1` vs `1.14.36.1`)

Reply:
would `NPS_VER=1.14.33.1-RC1` work for you? There are no other versions for 1.18.0+ nginx, everything else is >2018 or older. 

`1.14.36.1` is the Apache2 client, this version is the only recent official release